draco/types.py

```diff
@@ -1,3 +1,315 @@
-from typing import Iterable, TypeAlias
+from typing import Iterable, Literal, TypeAlias
 
+# granular imports to satisfy `pyright`
+import pydantic.class_validators as pydantic_validators
+import pydantic.config as pydantic_config
+import pydantic.fields as pydantic_fields
+import pydantic.main as pydantic_main
+import pydantic.types as pydantic_types
+
+"""
```

Comment:
As discussed, this file is Vega specific and should be moved into the Vega renderer.